readme.md

```diff
@@ -3,14 +3,26 @@ LÖVE is an *awesome* framework you can use to make 2D games in Lua. It's free,
 [![Build Status: Windows](https://ci.appveyor.com/api/projects/status/chc0hdr08wv1d5c7?svg=true)](https://ci.appveyor.com/project/AlexSzpakowski/love)
 [![Build Status: Github CI](https://github.com/love2d/love/workflows/continuous-integration/badge.svg)](https://github.com/love2d/love/actions?query=workflow%3Acontinuous-integration)
 
-Documentation
--------------
+Please see our [living wiki documentationwiki][wiki] for deeper explanations and documentation of this framework.
 
-We use our [wiki][wiki] for documentation.
-If you need further help, feel free to ask on our [forums][forums], our [Discord server][discord], or our IRC channel [#love on OFTC][irc].
+Come join us on our [forums][forums], our [Discord chat][discord], or our IRC channel [#love on OFTC][irc] and feel free to ask us anything.
 
-Compilation
------------
+Contributing
+------------
+
+Please contribute by looking through the [open issues][issues] and the forums and chat.
```

Comment:
The phrasing here kind of misses the original intent of what was there before, I think. Posting an issue and talking about potential changes in the forums and chat channels are a form of contribution. Replying to an existing issue in the tracker is as well, and I think the original text covers all of those cases.

The original phrasing kind of makes it seem like pull requests aren't a part of the above "best places" to contribute so a change to that phrasing might be helpful. As zorg says, the new phrasing is weird though.

 Also I'm not sure whether it should be explicitly pointed out in the readme or not, but for any large-scale or API-related change, discussing it through the forums/chat or the issue tracker is usually considered good manners before any pull request is made.

Reply:
Right -- I've changed back so the first sentence is as it was, just with an added direct link to the open issues page.
The second sentence is now changed from

For code contributions, pull requests and patches are welcome. Be sure to read the [source code style guide][codestyle].

To:

Ready to make a code contribution? Excellent! Please come discuss any bigger changes in the chat, and do also check our [source code style guide][codestyle].